string RA
string DEC
---
TelescopeInfo status
